### PR TITLE
[KYV-749] Enable payment exp api to fetch paytrail payment methods with core payment api

### PR DESCRIPTION
**Pull request by martinldev**
Enable payment exp api to fetch paytrail payment methods with core payment api

### CODE REVIEW

File: packages/configuration-backend/src/service.ts
Comment (martinldev) [inline packages/configuration-backend/src/service.ts:323]: This implementation is copied from your PR KYV-699. I will merge this after your PR has been merged, so I can sync the branches an then there will not be any conflicts.

File: packages/payment-backend/src/service.ts
Comment (severikupari1) [inline packages/payment-backend/src/service.ts:216]: Is this function call changed because you left out that .order?

**Review by severikupari1 — APPROVED** [review]
Looks good

**Pull request merged**
